File: ch04/NavigationBar/NavigationBar/ViewController.m
//
//  ViewController.m
//  NavigationBar
//
//  Created by MyZiyue on 2018/12/13.
//  Copyright © 2018 MyZiyue.com. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@property (weak, nonatomic) IBOutlet UILabel *label;

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
}
- (IBAction)save:(id)sender {
    self.label.text = @"点击Save";
}

- (IBAction)add:(id)sender {
    self.label.text = @"点击Add";
}

@end

File: ch04/SToolbar/SToolbar/ViewController.swift
//
//  ViewController.swift
//  SToolbar
//
//  Created by MyZiyue on 2018/12/13.
//  Copyright © 2018 MyZiyue.com. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    @IBOutlet weak var label: UILabel!
    
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
    }

    @IBAction func save(_ sender: Any) {
        self.label.text = "点击Save";
    }
    
    @IBAction func open(_ sender: Any) {
        self.label.text = "点击Open";
    }
}

